vue.config.js

```diff
@@ -1,3 +1,6 @@
 module.exports = {
   lintOnSave: false,
+  publicPath: process.env.NODE_ENV === 'production'
+    ? '/girder_web_components'
```

Comment:
This means that when doing `yarn build` to build the dist package, our `publicPath` will get set to `/girder_web_components`. Is that OK? What affect will that have on the bundle for downstreams (i.e. environments other than gh-pages)?

Reply:
I don't believe publicPath affects builds in `--target lib` mode.  

https://github.com/vuejs/vue-cli/issues/2196 -- looks like publicPath is overridden.

Reply:
Good to know. One thing that might be worth doing is putting a comment here on line 4 that says "this should only affect gh-pages deployment" so we know why it's here.